src/cljc/athens/common_events/schema.cljc

```diff
@@ -182,7 +182,24 @@
   [:map
    [:event/args
     [:map
-     [:unlinked-refs seq?]
+     [:unlinked-refs
+      [:sequential
+       [:vector
+        [:cat
+         string?
+         [:vector
+          [:map
+           [:block/string string?]
```

Comment:
This is whole block that we're transferring over the wire.
It's wrong because it contains a lot of irrelevant information, like `:db/id`.
Can we identify what we need from these blocks and see if it makes sense to just send that?

Reply:
Updated